src/server/lib/IDAPIFetch.ts

```diff
@@ -1,9 +1,20 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 
-import fetch, { RequestInit, Response } from 'node-fetch';
+import type { RequestInit, Response } from 'node-fetch';
```

Comment:
woah, honestly surprised that this is the only file where we're using node-fetch